get_and_write_play_by_play.py

```diff
@@ -0,0 +1,46 @@
+import pandas as pd
+from sqlalchemy.orm import sessionmaker
+
+from games import Games
+from models import Players
+from nba_api_client import NbaApiClient
+from play_by_play import PlayByPlay
+from player_comparator import PlayerComparator
+
+
+def clean_and_return_games_and_pbp_dfs(raw_games_df, raw_plays_dfs):
```

Comment:
Please add comments describing what the functions do.

As a bonus add type hinting to make the inputs/outputs explicit.